sdk/sdk.go

```diff
@@ -151,22 +94,25 @@ func (sdk *ShannonSDK) GetApplicationsDelegatingToGateway(
 // response after verifying the supplier's signature.
 func (sdk *ShannonSDK) SendRelay(
 	ctx context.Context,
-	sessionSupplierEndpoint *types.SingleSupplierEndpoint,
+	header *sessiontypes.SessionHeader,
```

Comment:
I prefer providing `types.SingleSupplierEndpoint` which could ensure matching `SessionHeader`, `SupplierAddress` and `EndpointUrl` instead of giving unnecessary responsibility to the consumer to collect/match and provide these independently.

Reply:
Thank you for the review. Updated, PTAL.